koku/api/resource_types/aws_ec2_compute_os/view.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2024 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""View for AWS accounts."""
+from django.conf import settings
+from django.db.models import F
+from django.utils.decorators import method_decorator
+from django.views.decorators.vary import vary_on_headers
+from rest_framework import filters
+from rest_framework import generics
+from rest_framework import status
+from rest_framework.response import Response
+
+from api.common import CACHE_RH_IDENTITY_HEADER
+from api.common.pagination import ResourceTypeViewPaginator
+from api.common.permissions.aws_access import AwsAccessPermission
+from api.resource_types.serializers import ResourceTypeSerializer
+from reporting.provider.aws.models import AWSCostEntryLineItemSummaryByEC2Compute
+
+
+class AWSEC2ComputeOperatingSystemView(generics.ListAPIView):
+    """API GET list view for AWS EC2 compute operating systems."""
+
+    queryset = (
+        AWSCostEntryLineItemSummaryByEC2Compute.objects.annotate(**{"value": F("operating_system")})
```

Comment:
Same here: I don't think unpacking is necessary.
```suggestion
        AWSCostEntryLineItemSummaryByEC2Compute.objects.annotate(value=F("operating_system"))
```

Reply:
I've update to remove unpacking